Rakefile

```diff
@@ -36,7 +46,7 @@ task :clean do
   sh "xcodebuild clean -configuration Release -scheme #{SCHEME} -destination '#{DESTINATION}' | xcpretty -c; exit ${PIPESTATUS[0]}"
 end
 
-task :default => ["test:specs", "test:demo"]
+task :default => ["test:package", "test:specs", "test:demo"]
```

Comment:
I ended up making a new `test` action that's the default, and adding `test:all` for CI to run.